src/config.ts

```diff
@@ -1,34 +1,34 @@
 import { constants as fhirConstants } from '@projecttacoma/node-fhir-server-core';
-import env from 'var';
+import 'dotenv/config';
+import * as env from 'env-var';
 
 // Set up whitelist
-const whitelist_env =
-  (env.WHITELIST && env.WHITELIST.split(',').map((host: string) => host.trim())) || false;
+const whitelist_env = env.get('WHITELIST').asArray() || false;
 
 // If no whitelist is present, disable cors
 // If it's length is 1, set it to a string, so * works
 // If there are multiple, keep them as an array
 const whitelist = whitelist_env && whitelist_env.length === 1 ? whitelist_env[0] : whitelist_env;
 export default {
   server: {
-    port: env.PORT || env.SERVER_PORT,
+    port: env.get('PORT').asInt() || env.get('SERVER_PORT').asInt(),
```

Comment:
Should `PORT`'s default value (whether that's set by the user or by REMS's Node app default config) also be listed in in .env.example?

Reply:
done